docs/06-concepts/04-exceptions.md

```diff
@@ -50,3 +50,35 @@ catch(e) {
   print('Something else went wrong.');
 }
 ```
+
+### Default Values in Exceptions
+
+Serverpod supports the use of default values for fields in exceptions, just like it does for models. This feature allows you to define default values for any supported field type directly within your exception definitions.
```

Comment:
Suggestion: Iterate this with GPT.

This sentence feels a bit strange; how about running it through ChatGPT a couple of times to improve it?

Reply:
We could probably already refer to the models section in the introduction but then specify that only `default` and `defaultModel` are supported.